src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/Models/OutputFieldMappingEntry.cs

```diff
@@ -0,0 +1,77 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Search.Models
+{
+    using Microsoft.Rest;
+    using Newtonsoft.Json;
+    using System.Linq;
+
+    /// <summary>
+    /// output field mapping for a skill.
+    /// </summary>
+    public partial class OutputFieldMappingEntry
+    {
+        /// <summary>
+        /// Initializes a new instance of the OutputFieldMappingEntry class.
+        /// </summary>
+        public OutputFieldMappingEntry()
+        {
+            CustomInit();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the OutputFieldMappingEntry class.
+        /// </summary>
+        /// <param name="name">The name of the output.</param>
+        /// <param name="targetName">The target name of the output.</param>
```

Comment:
It's more of a path than a name, isn't it?

Reply:
Rewording to "The name path of the output"